interval_hash.go

```diff
@@ -34,6 +35,26 @@ func (h *IntervalHash) Get(rc redis.Conn, field string) (string, error) {
 	return value, nil
 }
 
+//go:embed lua/ihash_mget.lua
+var ihashMGet string
+var ihashMGetScript = redis.NewScript(-1, ihashMGet)
+
+// MGet returns the values of the given fields
+func (h *IntervalHash) MGet(rc redis.Conn, fields ...string) ([]string, error) {
```

Comment:
However the naming here MGET could be confused with the REDIS MGET.

Reply:
`IntervalHash.Get` is equivalent to `HGET`, `IntervalHash.Set` is equivalent to `HSET` etc so I'm continuing with that pattern